src/app/pages/home/onboarding/index.js

```diff
@@ -0,0 +1,160 @@
+						<div className="bluehost-onboarding-step__image">
+							<img src={ comment } alt="" />
+						</div>
+						<div className="bluehost-onboarding-step__title">
+							{ __('Start with a page or post', 'bluehost-wordpress-plugin') }
+						</div>
+						<p className="bluehost-onboarding-step__description">
+							{ __("Use a page for content that won't change too often (your homepage, portfolio), and use posts for information that's timely (blog posts, updates).", 'bluehost-wordpress-plugin') }
+						</p>
+						<div className="bluehost-onboarding-step__actions">
+							<Button href={ `${ baseUrl }post-new.php` } isLink>
+								{ __('Add a blog post', 'bluehost-wordpress-plugin') }
+							</Button>
+							<Button href={ `${ baseUrl }post-new.php?post_type=page` } isSecondary>
+								{ __('Add a page', 'bluehost-wordpress-plugin') }
+							</Button>
+						</div>
+					</div>
+
+					<div className="bluehost-onboarding-step">
+						<div className="bluehost-onboarding-step__image">
+							<img src={ computer } alt="" />
+						</div>
+						<div className="bluehost-onboarding-step__title">
+							{ __('Make it look just right', 'bluehost-wordpress-plugin') }
+						</div>
+						<p className="bluehost-onboarding-step__description">
+							{ __("Use the customizer to go simple, bold—whatever's you.", 'bluehost-wordpress-plugin') }<br />
+							{ __("Or browse themes and try 'em out.", 'bluehost-wordpress-plugin') }
+						</p>
+						<div className="bluehost-onboarding-step__actions">
+							<Button href={ `${ baseUrl }admin.php?page=bluehost#/marketplace/themes` } isLink>
+								{ __('Browse themes', 'bluehost-wordpress-plugin') }
+							</Button>
+							<Button href={ `${ baseUrl }customize.php` } isSecondary>
+								{ __('Customize your site', 'bluehost-wordpress-plugin') }
+							</Button>
+						</div>
+					</div>
+
+					<div className="bluehost-onboarding-step">
+						<div className="bluehost-onboarding-step__image">
+							<img src={ rocket } alt="" />
+						</div>
+						<div className="bluehost-onboarding-step__title">
+							{ __('Get ready to launch', 'bluehost-wordpress-plugin') }
+						</div>
+						<p className="bluehost-onboarding-step__description">
+							{ __('Your site has a \"coming soon\" message, which lets people know you\'re working on it. ', 'bluehost-wordpress-plugin') }
+							<span
+								dangerouslySetInnerHTML={
+									{
+										__html: sprintf(
+											__('Check out our %swebsite pre-publishing%s checklist to help get ready for launch.', 'bluehost-wordpress-plugin'),
+											'<a href="https://www.bluehost.com/help/article/website-publish-checklist">',
+											'</a>'
+										)
+									}
+								}
+							/>
+
+						</p>
+						<div className="bluehost-onboarding-step__actions">
+							<Button onClick={ launchHandler } isPrimary>
+								{ __('Launch your site', 'bluehost-wordpress-plugin') }
+							</Button>
+						</div>
+					</div>
```

Comment:
We should wrap this in a check of the `comingSoon` setting to make sure that Coming Soon setting is actually on. Otherwise the text doesn't make sense.

Reply:
Added